cdap-ui/app/cdap/components/ThemeWrapper/index.tsx

```diff
@@ -30,24 +31,9 @@ export default class ThemeWrapper extends React.PureComponent<IThemeWraperProps>
       primary: {
         main: '#1a73e8',
       },
-      blue: {
-        50: '#045599',
-        100: '#0076dc',
-        200: '#0099ff',
-        300: '#58b7f6',
-        400: '#7cd2eb',
-        500: '#cae7ef',
-      },
-      grey: {
-        50: '#333333',
-        100: '#666666',
-        200: '#999999',
-        300: '#bbbbbb',
-        400: '#cccccc',
-        500: '#dbdbdb',
-        600: '#eeeeee',
-        700: '#f5f5f5',
-      },
+      blue,
+      grey,
+      green,
```

Comment:
If we have moved all the colors can we just add them to our theme (red, bluegrey, orange etc.,)

Reply:
Done!